### DISCUSSION TITLE
Provide user guides

### DISCUSSION THREAD

**Discussion by Mosnar** [craftcms/console#68] [Craft Console]
While Craft Console may be intuitive to us (developers) - it would immensely helpful to have a number of step-by-step user guides and videos we can send to non-technical people, such as billing managers and clients. Here are some ideas for topics:

**How to..**
- accept an organization invite and add billing information
- renew expired licenses
- use purchase approval
- resolve license warnings in the CMS
- setup a new organization and add a developer to help purchase licenses
- transfer a license from a personal account to an organization
- associate a plugin license with a CMS license
- use developer support plans

**Comment by brandonkelly** [comment]
Agree we should!

**Comment by AugustMiller** [comment]
We've rolled out a number of articles over the last year for developers + clients:

- [What is Craft Console?](https://craftcms.com/knowledge-base/what-is-craft-console)
- [Using Organizations in Craft Console](https://craftcms.com/knowledge-base/craft-console-organizations)
- [Setting up an Organization for your Developer](https://craftcms.com/knowledge-base/craft-console-for-clients)
- [Finding your Craft License Key](https://craftcms.com/knowledge-base/finding-your-craft-license-keys)
- [Releasing and Claiming Licenses](https://craftcms.com/knowledge-base/releasing-and-claiming-licenses)
